### PR TITLE
Automatic handling of versions with versioneer

### PR DESCRIPTION
**Pull request by observingClouds**
In preparation of #14 I propose to use [versioneer](https://github.com/python-versioneer/python-versioneer) for automatically determining the package version from its git commit.

So publishing a release will be as easy as described [here](https://github.com/python-versioneer/python-versioneer/blob/master/INSTALL.md#post-installation-usage)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -52,10 +54,6 @@ def run(self):
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
     ],
-    cmdclass={
-        "develop": PostDevelopCommand,
-        "install": PostInstallCommand,
-    },
```
Comment (aaronspring) [inline setup.py:58]: needed to install julia packages

File: versioneer.py [hunk 2]
```diff
@@ -0,0 +1,2189 @@
+# Version: 0.22
```
Comment (aaronspring) [inline versioneer.py:1]: do we really need this?

**Comment by aaronspring** [comment]
I find setuptools as in https://github.com/xarray-contrib/xskillscore/blob/main/setup.py much easier, also https://github.com/pydata/xarray/blob/main/setup.py

**Comment by aaronspring** [comment]
maybe we should just copy a working version from another project. releases can even be done with `GHA`

File: versioneer.py (hunk 2 above)
Comment (observingClouds) [reply versioneer.py:1]: Yes :P I mean, we could also look for the functions that we actually use for our use case but normally you would add this whole file.

**Pull request closed without merging**